nephos/helpers/misc.py

```diff
@@ -17,43 +18,41 @@
 t = Terminal()
 
 
-def execute(command, verbose=False, show_command=True, show_errors=True):
+def execute(command, show_response=True, show_command=True, show_errors=True):
```

Comment:
Having the logging module now, can't we remove show_response, show_command and show_errors and the logic for those flags, and just print the different levels of logging, will make this method quite more compact and clear.